Automattic/jetpack
# JITM styling for secondary button

Fixes #32345 - the JITM component didn't look to have `is-secondary` nor any secondary styling. Have added this in line with the style library.

## Proposed changes:
* Adds a `is-secondary` button class to the JITM if a second button is shown
* Styles the button in line with Emerald components.

## Jetpack product discussion
Noticed as part of testing the Newsletter JITM - discussed via slack.

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Go to the `/wp-admin/edit.php` endpoint
* The JITM should show the external link on Newsletter JITM (you may need to dismiss Blaze first, then wait 3 minutes due to Throttling of JITMs)
